.golangci.yaml

```
@@ -1,234 +1,239 @@
+        - stringXbytes
+        - switchTrue
+        - typeAssertChain
+        - typeSwitchVar
+        - typeUnparen
+        - underef
+        - unlambda
+        - unnecessaryBlock
+        - unslice
+        - valSwap
+        - weakCond
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)